sklearn/neural_network/tests/test_mlp.py

```diff
@@ -458,25 +458,25 @@ def test_tolerance():
     # It should force the algorithm to exit the loop when it converges.
     X = [[3, 2], [1, 6]]
     y = [1, 0]
-    clf = MLPClassifier(tol=0.5, max_iter=3000, algorithm='sgd', verbose=10)
+    clf = MLPClassifier(tol=0.5, max_iter=3000, algorithm='sgd')
     clf.fit(X, y)
     assert_greater(clf.max_iter, clf.n_iter_)
 
 
-def test_verbose_sgd():
+def test_verbose():
     # Test verbose.
     X = [[3, 2], [1, 6]]
     y = [1, 0]
-    clf = MLPClassifier(algorithm='sgd', max_iter=2, verbose=10,
-                        hidden_layer_sizes=2)
-    old_stdout = sys.stdout
-    sys.stdout = output = StringIO()
-
-    clf.fit(X, y)
-    clf.partial_fit(X, y)
-
-    sys.stdout = old_stdout
-    assert 'Iteration' in output.getvalue()
```

Comment:
why did you removed the assertion?
We probably want to test that verbose has an effect